db/fixtures/miq_product_features.yml

```diff
@@ -3419,7 +3428,7 @@
       - :name: Edit Tags
         :description: Edit Tags of PXE Servers
         :feature_type: control
-        :identifier: pxe_tag
+        :identifier: pxe_server_tag
```

Comment:
Similarly here - if we can change the ui code to use the old name first, then we can change this later with a db migration.

Reply:
Note, this fetaure didn't exist before this PR.  I will clean/merge up the commits.  I just have to coordinate with @kavyanekkalapu since she's using my branch.

Reply:
Ok, squashed.  Now, it's obvious that this is a new feature.